matsim/src/main/java/org/matsim/api/core/v01/events/LinkEnterEvent.java

```diff
@@ -35,31 +35,29 @@
 	
 	private final Id<Link> linkId;
 	private final Id<Vehicle> vehicleId;
+	private final Id<Person> driverId;
```

Comment:
In this class, driverId is re-introduced although it was deprecated before (see method `getDriverId()`). I know we discussed that it might help in some event handlers for improved performance, but at the moment it's not yet used. Do we really want to re-add the driver id, or should we tackle the performance problems in events handler differently? I can't remember what we decided here.

Reply:
Good point Marcel. We can leave these changes outside this PR since they are not being used. Later we can assess the performance benefits if we decide to change the implementation of handlers to improve performance. I will update the PR.